gemma-core/src/main/java/ubic/gemma/persistence/util/AclQueryUtils.java

```diff
@@ -72,6 +73,10 @@ public class AclQueryUtils {
     //language=SQL
     static final String ANONYMOUS_SID_SQL = "select sid.ID from ACLSID sid where sid.GRANTED_AUTHORITY = 'IS_AUTHENTICATED_ANONYMOUSLY'";
 
+    public static String formAclRestrictionClause( String aoiIdColumn ) {
```

Comment:
It should be mentioned in the javadoc that this is passing `BasePermission.READ`.